src/CMake/findpackage.cmake

```diff
@@ -0,0 +1,60 @@
+message("-- Preparing XRT find_package")
```

Comment:
This file is black magic to me.  Can you add a few comments?  For example, it does look like it scales very well, because if  new targets are added, then this file must apparently be updated.

Reply:
I changed it to be more scalable